src/docs/installing-a-coding-environment/2-install-tools-mac.md

```diff
@@ -65,43 +72,48 @@ Now, type:
 brew install git
 ```
 
-Likewise, when you are done, from your node.js command prompt window, type: 
+Likewise, when you are done, from the same command prompt window, type:
 
 ```
 git --version
 ```
 
-and veryify that it's installed. 
+and veryify that it's installed and working.
 
 ## Get a Mapbox token
 
-If you are going to follow a tutorial that uses React-Map-GL, you will probably need a Mapbox token. Go to [Mapbox.com](http://mapbox.com) and sign up or sign in, then go to your account, and finally, click on API access tokens. 
+If you are going to follow a tutorial that uses React-Map-GL, you will also need a Mapbox token. Go to [Mapbox.com](http://mapbox.com) and sign up or sign in, then go to your account, and finally, click on [API access tokens](https://www.mapbox.com/studio/account/tokens/).
 
 Either create a new token or copy the one you want to use.
 
-Next, we're going to put that token in your .bash_profile file so that you can access it when needed. 
-
-.bash_profile doesn't necessarily exist, on a new machine, it has to be created. So either find it (in your home folder) or create a new file. 
+Next, we're going to put that token in your `.bash_profile` file so that your application can access it.
 
-There, add the line: 
+From the command line, run:
 
 ```
-export MAPBOX_TOKEN="[your mapbox access token]"
+echo 'export MapboxAccessToken="[your mapbox access token]' >> $HOME/.bash_profile
```

Comment:
Maybe precise that you if you are using another shell you should append the export to the appropriate file?

Reply:
I feel like if you're using a shell other than bash you know what this does.

We could also just add this to ~/.profile to be agnostic.

Reply:
Fair enough haha

I'm not sure everyone has one, you go back to the issue of sourcing it from the right shell config ;)